src/api/Feed.js

```diff
@@ -403,8 +405,7 @@ class Feed extends Base {
         try {
             await Promise.all(
                 task.addedAssignees.map(assignee =>
-                    // $FlowFixMe
-                    assignee.type === 'group'
+                    assignee.item && assignee.item.type === 'group'
```

Comment:
It seems odd that the the prior logic didn't cause any tests to fail.

Reply:
The previous code path was not connected to the API (this PR does that), so tests for it used mock data matching the expected shape. The assumption was that the selector object would be equivalent to {...item} + other keys, but this refactoring uses the safer pattern by accessing the `item` instead. The test data has `type` for item already.